documentation/src/main/asciidoc/reference/schema.asciidoc

```diff
@@ -258,3 +258,82 @@ on an AWS Elasticsearch cluster.
 The only workaround is to avoid the schema update on AWS.
 It should be avoided in production environments regardless:
 see <<schema-management-concepts-update-failure>>.
+
+
+[[schema-management-export]]
+== Exporting the schema
+
+[[schema-management-export-filesystem]]
+=== Exporting the schema to a set of files
+
+[WARNING]
+====
+The schema export is backend specific.
+Only the <<backend-elasticsearch,Elasticsearch backend>> provides the information required to create an index.
+The <<backend-lucene,Lucene backend>> does not provide any actual schema information beyond the list of indexes.
+====
+
+Hibernate Search ships with a built-in schema collector that writes the content of a schema export to a disk.
+All indexes represented by a `SearchSchemaManager` will be written to their own directory as an `index.json` file containing
+information on the index. The output is backend specific.
```

Comment:
The file name is obsolete... But this needs to be reworded anyway as we didn't explain schema collectors at this point, and the user actually doesn't need to know we use a schema collector under the hood:

```suggestion
The <<mapper-orm-schema-management-manager,schema manager>> provides a way to export schemas to the filsystem. The output is backend-specific.

[NOTE]
====
Schema exports are constructed based on the mapping information and configurations (e.g. such as the backend version).
Resulting exports are not compared to or validated against the actual backend schema.
====
```